setup.py

```diff
@@ -103,7 +103,7 @@
 
 MAJOR = 0
 MINOR = 0
-MICRO = 15
+MICRO = 16
```

Comment:
`pylint` complains: `setup.py:107:10: C0209: Formatting a regular string which could be a f-string (consider-using-f-string)`. This is an optional change if the developer has time.

Reply:
Fixed.